assembl/static2/js/app/components/administration/saveButton.jsx

```diff
@@ -339,6 +378,7 @@ const mapStateToProps = ({
     thematics: thematicsInOrder.toArray().map(id => thematicsById.get(id).toJS()),
     preferences: discussionLanguagePreferences,
     i18n: i18n,
+    debate: debate,
```

Comment:
if it is just for timeline, you could just pass `debate.debateData.timeline` in props

Reply:
yes